pkg/agent/datamodel/sig_config.go

```diff
@@ -477,6 +481,20 @@ var (
 		Version:       LinuxSIGImageVersion,
 	}
 
+	SIGCBLMarinerV2FIPSGen1ImageConfigTemplate = SigImageConfigTemplate{
```

Comment:
```suggestion
	sigCBLMarinerV2FIPSGen1ImageConfigTemplate = SigImageConfigTemplate{
```

probably would make lint happy without breaking everything? unsure if it's used elsewhere

Reply:
Made the change but looks like we still have the golang lint errors: sigCBLMarinerV2FIPSGen1ImageConfigTemplate is a global variable (gochecknoglobals)